php/classes/handlers/class-castos-handler.php

```diff
@@ -346,7 +346,7 @@ public function upload_podcast_to_podmotor( $post ) {
 
 		$this->logger->log( 'Upload Podcast Response', $response_object );
 
-		if ( ! isset( $response_object->status ) || ! $response_object->status ) {
+		if ( ! isset( $response_object->success ) || ! $response_object->success ) {
```

Comment:
Fixed a bug - $response_object->status is integer, and it returns an integer even if error occurs. So we need to check $response_object->success instead.